src/ESPEasy_checks.ino

```diff
@@ -34,7 +34,7 @@ template<typename T, typename U> constexpr size_t offsetOf(U T::*member)
 void run_compiletime_checks() {
   check_size<CRCStruct,                             168u>();
   check_size<SecurityStruct,                        593u>();
-  const unsigned int SettingsStructSize = (244 + 82 * TASKS_MAX);
+  const unsigned int SettingsStructSize = (252 + 82 * TASKS_MAX);
```

Comment:
244+6

Reply:
Actually it passes after 252, not 260 :/
But yes, you've got right it shou;d be under HAS_ETHERNET !